crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/switchboard_state.rs

```diff
@@ -543,22 +543,23 @@ impl SwitchboardState {
                 window.contains(&op.timestamp)
                     // Does the op fall within one of the specified arcsets
                     // with the correct integration/limbo criteria?
-                        && agents.iter().fold(false, |yes, (agent, arc_set)| {
-                            if yes {
-                                return true;
-                            }
-                            arc_set.contains((**op_loc8).into()) &&
-                            self.local_agent_by_hash(agent)
-                                .and_then(|agent| {
-                                    agent
-                                        .ops
-                                        // Does agent hold this op?
-                                        .get(op_loc8)
-                                        // Does it meet the limbo criteria of the query?
-                                        .map(|op| include_limbo || op.is_integrated)
-                                })
-                                .unwrap_or(false)
-                        })
+                        && arc_set.contains((**op_loc8).into())
+                // && agents.iter().fold(false, |yes, (agent, arc_set)| {
```

Comment:
commented code

Reply:
Above all fixed in `140576f89e08d81a09e9c21a2a84d9d17a66482b`

Reply:
Fixed in https://github.com/holochain/holochain/commit/140576f89e08d81a09e9c21a2a84d9d17a66482b